cmd/restic/cmd_restore.go

```diff
@@ -219,6 +232,31 @@ func runRestore(ctx context.Context, opts RestoreOptions, gopts GlobalOptions,
 		res.SelectFilter = selectIncludeFilter
 	}
 
+	if !hasXattrExcludes && !hasXattrIncludes {
+		// set default of including xattrs from the 'user' namespace
+		opts.IncludeXattrPattern = []string{"user.*"}
```

Comment:
I was a bit wary about changing the default - For the moment (after moving to a separate func and updating the windows implementation) I ended up making the default to restore all - partly because I'm not sure the best way to have a separate default for linux only.  Any advice on setting this for linux only? (assuming we still want user.* as a default)

Reply:
To run code on Linux only, you could check for `runtime.GOOS=="linux"